<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spo2Window</class>
 <widget class="QWidget" name="Spo2Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPO2 Window</string>
  </property>
  <widget class="QPushButton" name="pExit_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>99</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pClear_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>99</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QFrame" name="test">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>freqence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>timeout</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>readcount</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pFre_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="pTm_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="pRc_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pFreOk_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pRcCancel_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pFreCancel_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pRcOk_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pST_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>start to test</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pStatistics_txt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pShowReadData_check">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>show read data</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="pMsg_Txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="pGenerateData_check">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>generate new data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pConnect_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pSendTestData_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>sendTestData</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pDisConnect_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
